src/jit/block.h

```diff
@@ -1090,7 +1090,8 @@ struct BasicBlock : private LIR::Range
 
     // Clone block state and statements from 'from' block to 'to' block.
     // Assumes that "to" is an empty block.
-    static void CloneBlockState(Compiler* compiler, BasicBlock* to, const BasicBlock* from);
+    static bool CloneBlockState(
```

Comment:
Update the header to the standard one, and detail what happenes with the return value